MekHQ/src/mekhq/campaign/rating/FieldManualMercRevDragoonsRating.java

```diff
@@ -314,10 +307,10 @@ private void updateDropJumpShipSupportNeeds(Entity en) {
     //   3 + (4/5) = 3 + 0.8 = 3.8 = 4 hours.
     //   total = 16 hours.
     private void calcMedicalSupportHoursNeeded() {
-        int numSquads = new BigDecimal(getCampaign().getPersonnel().size())
+        int numSquads = new BigDecimal(getCampaign().getActivePersonnel().size())

```

Comment:
So, one more comment: since "getActivePersonnel()" can get expensive for a lot of personnel, you should consider storing that count locally rather than calling it multiple times.

The alternatives solution is to store the number of active personnel locally within Campaign, but then you'd have to update it when the personnel count changes.

Reply:
Hmm I'd prefer option 2 really, but need to make sure Person is firing events for any method that changes their status first.